lib/translator/unmap.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_TRANSLATOR_UNMAP_H
+#define LIB_TRANSLATOR_UNMAP_H
+
+#include "common.h"
+
+#include "third_party/spdk/nvme.h"
+
+namespace translator {
+
+StatusCode UnmapToNvme(Span<const uint8_t> scsi_cmd,
```

Comment:
can we add a comment here describing what nvme command unmap corresponds to and maybe also describing what buffer_out is?

Reply:
I'll go ahead and add a comment about buffer_out. The unmap NVMe equivalent is linked in the .cc file. Should we put it here as well?

Reply:
Oh I see the comment in the .cc. I think it'd be helpful in the header file to say that Unmap translates to an NVMe Dataset Management command